app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -28,10 +28,10 @@
     private final int heightPixels;
     private final int widthPixels;
     private final boolean candyWordFirst;
-    private final String nidoFemale;
-    private final String nidoMale;
+    private final String nidoFemale = "Nidoran♀";
+    private final String nidoMale = "Nidoran♂";
```

Comment:
I'll see what I can do. The only translation for the Nido family is from English to Japanese (or generally non-latin character languages).